content/about/notes/1.1.1/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Istio 1.1.1
+publishdate: 2019-03-22
+icon: notes
+---
+
+Istio 1.1.1 is the first in a weekly series of patch releases for Istio 1.1.  It contains a small number of bug fixes and minor enhancements.
```

Comment:
I don't think we should commit to this being weekly.

Also, can we use the same text as in previous patch releases? which is:

This release includes security vulnerability fixes and improvements to robustness.
This release note describes what's different between Istio 1.1.1 and Istio 1.0.